### PR TITLE
feat: initial heat map

### PR DESCRIPTION
**Pull request by chungwwei**
it's the example code from mapbox with a little modification from me. Since it's using mapbox, you need to create a mapbox account and create a token, and create a `.env.local` file in `src` and put the mapbox token there to see map.

I don't really know how to style the heatmap in `MapStyle.ts`, doesn't look easy to me.

### CODE REVIEW

**Comment by kevmo314** [comment]
> I don't really know how to style the heatmap in `MapStyle.ts`, doesn't look easy to me.

What did you try? This would make a great mini Thursday talk :)

**Comment by juaoose** [comment]
Would the Mapbox access token be in the build config? Like in the stream-dashboard project secrets?

**Comment by kevmo314** [comment]
Nah the mapbox token can be shared publicly so it can be in the source

File: src/component/GlobalHeatMap.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { useState, useEffect, useMemo } from "react";
+import { render } from "react-dom";
+import MapGL, { Source, LayerProps, Layer } from "react-map-gl";
+import { heatmapLayer } from "./MapStyle";
+// import ControlPanel from './control-panel';
+
+const MAPBOX_TOKEN = ""; // Set your mapbox token here
```
Comment (juaoose) [inline src/component/GlobalHeatMap.tsx:8]:
```suggestion
const MAPBOX_TOKEN = "pk.eyJ1Ijoia2V2bW8zMTQiLCJhIjoiY2t3bjR3cTloMDJ1ajJ1cW9obGh2ZmcybCJ9.uWdEuy9ilDupIiaOQIcMpQ"; 
```

**Review by juaoose — CHANGES_REQUESTED** [review]

**Review by juaoose — APPROVED** [review]

**Pull request merged**